packages/jtex/src/imports.ts

```diff
@@ -1,17 +1,9 @@
 import type { TemplateImports } from './types.js';
+import { writeTexLabelledComment } from 'myst-common';
 
 const commentLenth = 50;
```

Comment:
Thanks @andry-tino -- just looking over this briefly, excited to get them in!

Saw this typo from before go to a few other files, would be great to clean it up, but not critical. :)

```suggestion
const commentLength = 50;
```